examples/gallery/plot/double-y-axes.py

```diff
@@ -0,0 +1,70 @@
+"""
+Double Y-axes
+-------------
+
+The ``frame`` parameter of the plotting methods of the :class:`pygmt.Figure`
+class can control which axes should be plotted and possibly show annotations
+and tick marks. By default, all the 4 axes are plotted, along with annotations
+and tick marks (denoted **W**, **S**, **E**, **N**). Lower cases (**w**, **s**,
+**e**, **n**) can be used to denote to only plot the axes with tick marks. We
+can also only plot the axes without annotations and tick marks using **l**
+(left axis), **r** (right axis), **t** (top axis), **b** (bottom axis).
+"""
+
+import numpy as np
+import pygmt
+
+# Generate common x values and two kinds of y values
+x = np.linspace(1.0, 9.0, num=9)
+y1 = x
+y2 = x ** 2 + 110
+
+fig = pygmt.Figure()
+
+# Plot y1
+# The left (W) and bottom (S) axes are plotted with annotations and tick marks
+fig.basemap(
+    region=[0, 10, 0, 10],
+    projection="X5c/5c",
+    frame=["WS", "xaf+lx", "yaf+ly1"],
+)
+# Plot the line for y1
+fig.plot(
+    x=x,
+    y=y1,
+    pen="1p,blue",
+)
```

Comment:
It's unnecessary to put arguments in separate lines, especially for lines no longer than 88 characters. What I usually do is writing all arguments in one single line, and let `black` (i.e., run `make format`) do the formatting automatically.
```suggestion
# Plot the line for y1
fig.plot(x=x, y=y1, pen="1p,blue")
```